network_visualizer/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(network_visualizer)
+find_package(catkin REQUIRED)
```

Comment:
Should include:
```
catkin_package()
catkin_python_setup()
```

Should also include
```
install(PROGRAMS scripts/open_network_visualizer
        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
)
```
but see my other comment about the structure of this package.